addons/binding/org.openhab.binding.homematic/README.md

```diff
@@ -402,6 +410,28 @@ sendCommand(Var_1, RefreshType.REFRESH)
 
 **Note:** adding new and removing deleted variables from the GATEWAY-EXTRAS Thing is currently not supported. You have to delete the Thing, start a scan and add it again. 
 
+**Deprecated configuration for Thing ...: no address property available. Using old style configuration, please update your things and items file(s)**
```

Comment:
Could you then also fill https://github.com/openhab/openhab-distro/wiki/Draft-for-openHAB-2.2-Release-Notes#homematic-binding with some infos? Thanks!